src/openrct2/Game.cpp

```diff
@@ -794,8 +671,7 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
     else if (
         command == GAME_COMMAND_SET_SCENERY_COLOUR || command == GAME_COMMAND_SET_WALL_COLOUR
         || command == GAME_COMMAND_SET_LARGE_SCENERY_COLOUR || command == GAME_COMMAND_SET_BANNER_COLOUR
-        || command == GAME_COMMAND_SET_BANNER_NAME || command == GAME_COMMAND_SET_SIGN_NAME
-        || command == GAME_COMMAND_SET_BANNER_STYLE || command == GAME_COMMAND_SET_SIGN_STYLE)
+        || command == GAME_COMMAND_SET_SIGN_NAME || command == GAME_COMMAND_SET_BANNER_STYLE)
```

Comment:
Sign name is game action and can therefore be removed

Reply:
Removed!